Gandheesh/1.Basics.html

```diff
@@ -0,0 +1,62 @@
+<html>
+
+<head>
+     <meta charset="UTF-8">
+     <meta name="description" content="Html basics">
+     <meta name="viewport" content="width=device-width,intial-scale=1.0">
+     <title> HTML Basics </title>
+</head>
+
+<body>
+     <h1> Merlo J. Pusey</h1>
+     <hr>
+     <p><cite> Wikipedia, the free encyclopedia</cite>
+     </p>
+     <img src="P1.jpg" width="400px" height="300px">
+     <p><b> Merlo John Pusey</b>(February 3, 1902, in Woodruff, Utah â€“ November 22, 1985, in Washington,
+          D.C.)<sup>[1][2]</sup> was an American biographer and editorial writer. He won the 1952 Pulitzer Prize for
+          Biography or Autobiography and the 1952 Bancroft Prize for his 1951 biography of U.S. Chief Justice Charles Evans Hughes.
+          <br><br>
+          Born and raised on a farm near Woodruff, Utah, Pusey was a Latter-day Saint. He attended the Latter-day Saints
+          University and graduated as a member of Phi Beta Kappa from the University of Utah after working on the
+          college newspaper. He later became
+          a reporter and assistant city editor at The Deseret News in Salt Lake City.[3]
+          <br><br>
+          <strike>Pussy</strike>Pusey worked for The Washington Post from 1928 to 1971, becoming associate editor in
+          1946, continuing to contribute occasional pieces until about two years before his death.</p>
+     <br><br>
+     <p><b>Contents</b>
+          <br>
+          
+          <ol>
+               <li><b><a href="#Books">Books</a></b></li>
+               <li><b><a href="#References">References</a></b></li>
```

Comment:
The links are references to their respective ids. In this case, these ids don't exist and hence the links are broken.
Create Books and References ids.